mongodb-odbc-driver/bin/build-mongo-odbc.sh

```diff
@@ -38,7 +38,12 @@
     if [ "$PLATFORM" = macos ]; then
         iODBC_dir=iODBC-3.52.12
         echo "downloading iODBC"
-        curl -O "http://noexpire.s3.amazonaws.com/sqlproxy/binary/linux/$iODBC_dir.tar.gz"
+        curl -O "http://noexpire.s3.amazonaws.com/sqlproxy/binary/linux/$iODBC_dir.tar.gz" \
```

Comment:
Since we are doing the same thing in multiple places, it might make sense to abstract the curl call into a shell function.